Documentation/guides/messages/README.md

```diff
@@ -27,11 +27,11 @@
 + [XA0033](xa0033.md): Failed to get the Java SDK version as it does not appear to contain a valid version number.
 + [XA0034](xa0034.md): Failed to get the Java SDK version. 
 
-+ [XA0100](xa0100.md): EmbeddedNativeLibrary is invalid in Android Application project. Please use AndroidNativeLibrary instead.
```

Comment:
Why is XA0100 and XA0104 removed?

It's certainly not related to "mov[ing] images for messages"...

Reply:
I've separated that into its own pull request now so the changes can be considered independently: <https://github.com/xamarin/xamarin-android/pull/3154>